facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -91,6 +91,7 @@ trait FapiFrontPress extends QueryDefaults with Logging with ExecutionContexts {
       .showElements("all")
       .showTags("all")
       .showReferences(references)
+      .pageSize(20)
```

Comment:
Does this still allow overrides? Also do the right people know this is happening, it will affect every container with the default size...